src/app/ReadClient.h

```diff
@@ -144,30 +144,31 @@ class ReadClient : public Messaging::ExchangeDelegate
      *  The ReadClient will automatically shut itself down when it receives a
      *  response or the response times out.  So manual shutdown is only needed
      *  to shut down a ReadClient before one of those two things has happened,
-     *  (e.g. if SendReadRequest returned failure).
+     *  (e.g. if SendReadRequestMessage returned failure).
      */
     void Shutdown();
 
     /**
      *  Send a Read Request.  There can be one Read Request outstanding on a given ReadClient.
-     *  If SendReadRequest returns success, no more Read Requests can be sent on this ReadClient
+     *  If SendReadRequestMessage returns success, no more Read Requests can be sent on this ReadClient
      *  until the corresponding InteractionModelDelegate::ReadDone call happens with guarantee.
      *
      *  @retval #others fail to send read request
      *  @retval #CHIP_NO_ERROR On success.
      */
-    CHIP_ERROR SendReadRequest(ReadPrepareParams & aReadPrepareParams);
+    CHIP_ERROR SendReadRequestMessage(ReadPrepareParams & aReadPrepareParams);
```

Comment:
Why are we renaming this?  The important thing is that we are sending a read request action.  Whether it's one message or not is implementation detail here.

Reply:
Rename it back, thanks